templates/index.html

```diff
@@ -32,7 +32,7 @@
   <span class="subheading">See all your Open Source contributions in one place</span>
   <input class="username js-username" type="text" autofocus id="username" placeholder="Enter your Github username" onkeypress="handleKeypress(event)"/>
   <button class="btn btn-primary" onclick="showPrs()">Show My Pr's</button>
-  <span class="madeBy">Made with ❤️ by <a href="https://twitter.com/geekykaran">geekykaran</a></span>
+  <span class="madeBy">Made with ❤️ by <a href="https://twitter.com/geekykaran">geekykaran</a>. You can find the source on <a href="https://github.com/karanjthakkar/showmyprs.com/">github</a></span>
```

Comment:
@cg-cnu Thanks so much for adding this. Definitely needed! 

Minor nits:
1. Change anchor text that says `github` -> `Github`
2. Remove trailing `/` from github link

Reply:
Sure thing. on it 😄